tests/capture/test_operators.py

```diff
@@ -0,0 +1,321 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Integration tests for the capture of pennylane operations into jaxpr.
+"""
+# pylint: disable=protected-access
+import pytest
+
+import pennylane as qml
+
+from pennylane.capture.meta_type import _get_abstract_operator
+
+jax = pytest.importorskip("jax")
+
+pytestmark = pytest.mark.jax
+
+AbstractOperator = _get_abstract_operator()
+
```

Comment:
In the tests I would add more simple circuits in qfunc, in order to make sure we are not missing anything.

Reply:
any particular edge cases come to mind @rmoyard?

Reply:
Not specifically but just have simple circuits that take some arguments including wires and return some quantum measurements and possibly some classical values as well. Just to make sure everything works as expected, I think basic tests need to be solid in order to avoid potential issues

Reply:
Measurement support will be a follow-up PR, as we still need to work out how and where to compute shape information.

 PR #5523  will extend testing to all the templates while fixing any issues the templates pose.